aether-common-module/aether/common/templates/aether/login.html

```diff
@@ -0,0 +1,132 @@
+{% extends "rest_framework/login_base.html" %}
+
+{% comment 'LICENSE' %}
+Copyright (C) 2018 by eHealth Africa : http://www.eHealthAfrica.org
+
+See the NOTICE file distributed with this work for additional information
+regarding copyright ownership.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with
+the License.  You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing,
+software distributed under the License is distributed on an
+"AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+KIND, either express or implied.  See the License for the
+specific language governing permissions and limitations
+under the License.
+{% endcomment %}
+
+{% load static %}
+
+{% block title %}{{ app_name }}{% endblock %}
+
+{% block meta %}
+  {% include 'aether/meta.html' %}
+  <link rel="shortcut icon" href="{% static 'aether/images/aether.png' %}"/>
+{% endblock %}
+
+{% block style %}
+  <link rel="stylesheet" type="text/css" href="{% static 'aether/css/styles.css' %}"/>
+
+  <style>
```

Comment:
likewise this, or maybe a separate `authentication.css`

Reply:
:+1: